packages/engine_umbrella/apps/engine/lib/vehicle.ex

```diff
@@ -19,6 +20,20 @@ defmodule Vehicle do
     :capacity
   ]
 
+  @hour_and_minutes_format ~r/^((?:[01]\d|2[0-3]):[0-5]\d$)/
+
+  validates(:start_address, presence: true)
+  validates(:earliest_start, format: [with: @hour_and_minutes_format, allow_nil: true])
+  validates(:latest_end, format: [with: @hour_and_minutes_format, allow_nil: true])
+
+  validates([:capacity, :weight],
+    by: [function: &Kernel.is_integer/1, message: "must be an integer"]
```

Comment:
Kernel. can be omitted here
```suggestion
    by: [function: &is_integer/1, message: "must be an integer"]
```

Reply:
And in other places where Kernel. is prepended.